### DISCUSSION TITLE
Content style guide

### DISCUSSION THREAD

**Discussion by tomyems** [UKHomeOffice/design-system#470] [General]
Add ideas, suggestions, improvements and corrections to the [content style guide](https://design.homeoffice.gov.uk/content-style-guide) to this discussion

**Comment by jbuller** [comment]
A few ideas
- DDaT as an acronym and in full?
- Hyphenation of 'user centred'? 
- Hashtags: Reference guidance on Pascal Case
- 'Horizon News' will be redundant soon as Sharepoint replaces Horizon.
- File extensions: PDF and DOCX or pdf and docx or 'Word'?
- Would be good to reference the GOV.UK style guide and any other internal style guides

**Reply by sshukor** [reply]
Thanks for your great suggestions @jbuller. I will document them and add them to our style guide backlog, which will be reviewed by the HO style council when we next meet in January. Cheers, Steve

**Reply by sshukor** [reply]
Update: Hi @jbuller,

We discussed your suggestions at our recent style council meeting.

- We think DDaT is a cross-gov term and as such I will reach out to GDS to ask them to consider adding it to their style guide.
- We will add a new entry for 'user centred' to the HO style guide
- We would like to explore the hashtags suggestion further. Please could you provide more information on your suggestion.
- We are checking timescales re Horizon News
- File extensions: could you provide some more information about the guidance you would like to see?
- We link to the GOV.UK style guide where relevant. Let me know if this is what you meant.

Thanks again for getting involved! You're helping to make the guide better for everyone.

**Reply by jbuller** [reply]
> * We would like to explore the hashtags suggestion further. Please could you provide more information on your suggestion.

https://www.accessible-social.com/copy-and-formatting/hashtags 
https://twitter.com/rnib/status/1128736809968492544

> * File extensions: could you provide some more information about the guidance you would like to see?

As I wrote should they be capitalised or program names used instead:
PDF and DOCX or pdf and docx or 'Word'?

> * We link to the GOV.UK style guide where relevant. Let me know if this is what you meant.

I see the references throughout. I meant a line at the top that says this supplements the GOV.UK style guide so see it for guidance on things not listed here.

**Reply by sshukor** [reply]
Thanks @jbuller. Some amusing hashtag bloopers there!

**Comment by robertdeniszczyc2** [comment]
Hi,

I have a couple pieces of feedback for consideration please:

Should the record for `biometric residence card` also have the note that `these are no longer in use` to reflect the synonymous entries of `UK residence card` and `EEA biometric residence card`?

And the following two entries seem to be contradictory:

```
His Majesty’s
Use ‘HM’ rather than ‘His Majesty’s’ in front of the name of the relevant public body. For example, ‘HM Revenue & Customs’.

His Majesty’s Passport Office
Capitalised. You can use ‘HM Passport Office’ after the first full mention. Do not use ‘HMPO’ in public-facing services.
```

The first one says to use `HM`, but the second says `use ‘HM Passport Office’ after the first full mention`; to write the first full mention in the second example would require breaking the guidance of the first if I'm reading it correctly?

Cheers,
Rob

**Reply by sshukor** [reply]
Thanks for your suggestions and highlighting these inconsistencies @robertdeniszczyc2. I will document them and add them to our style guide backlog, which will be reviewed by the HO style council on 27 January. Cheers, Steve

**Reply by sshukor** [reply]
Update: Hi Robert,
We discussed your suggestions and will be updating the style guide as follows:

- Update BRC, UK residence card and EEA BRC entries to make them consistent. Also, we are removing 'no longer in use' because it is not quite accurate. They are being phased out and can still be used in some cases.
- Update our HMPO guidance

Thanks again for getting involved. We really appreciate it. Keep the suggestions coming!

**Comment by TLStom** [comment]
Hi. I have a couple of terms and phrases that would be useful to confirm style:

- priority visa (PV) service / super priority visa (SPV) service: Should this be capitalised? Should it include "service"? What is the correct acronym, and when to use the acronym?
- When to use "Visa Application Centre (VAC)" or "UK Visa and Citizenship Application Services (UKVCAS)"
- When to use "settlement" vs "non-settlement" during the visa application process
- General capitalisation rules for various visa routes, for example "work", "study", "family", "high potential individual".
- Guidance on how to use UKVI.

**Reply by sshukor** [reply]
Hi @TLStom,

Thanks for these great suggestions! I'll take them back to the style council for discussion. In the meantime it would be good to know if you've had any feedback from users regarding these terms.

**Reply by sshukor** [reply]
Also:
- There is an existing entry on VAC. What is missing from your point of view?
- Could you provide some more information regarding the issue with using UKVI?

**Reply by sshukor** [reply]
Hi! We've just published new guidance on UKVCAS and VAC in the Home Office style guide following your suggestion. More to follow. Thanks again for getting involved!

**Comment by TLStom** [comment]
Thanks @sshukor. 

I'm noticing inconsistency between what's being used by external suppliers and the Home Office, so just want to align these. 

For the entry on VAC, it would just be good to know when to use the term "visa application centre" or "UK visa and citizenship application centre", so confirm that these refer to specific types of service points and are not interchangeable terms. 

For UKVI, it's on usage. First off, to confirm that it is "UK Visas and Immigration" and not "UK Visas & Immigration", and second when you can just use "UKVI" without using the full name.

**Reply by sshukor** [reply]
Thanks @TLStom. I'm following up on both of these excellent points. Thank you!

**Reply by sshukor** [reply]
See comment above. Thanks again!

**Reply by TLStom** [reply]
Thanks @sshukor!

**Reply by sshukor** [reply]
Quick update @TLStom, we've updated our guidance on UKVCAS and VAC. UKVI is still on our to do list.

**Comment by jbuller** [comment]
"Home Office" or "the Home Office"? (same for other departments and internal teams)
Seems like there's no easy answer: https://english.stackexchange.com/questions/389049/how-to-do-you-know-when-to-add-the-before-organisations-names
"XYZ team" rather than "XYZ Team"

**Reply by sshukor** [reply]
Thanks for this suggestion @jbuller. I'll follow that up.

**Comment by jbuller** [comment]
A quick summary of where my ideas have gone:
- GOV.UK has guidance on using lower case 'team' in team names.
- GOV.UK under Abbreviations has filenames in all capitals though does it in the context of saying not to expand them.
- Nothing on hashtags
- No word on '(the) Home Office'
- We at least have moved away from using but its still used elsewhere in government 'DDaT'

**Comment by jbuller** [comment]
A new suggestion. Something like:

**Access needs**
Primarily relating to the needs of users with temporary or permanent conditions or in situations that result in them being disabled. For example being unable to hear another persons voice effectively. This is a broader category than ‘disabled people’. It can also cover needs resulting from barriers such as language or lacking digital skills or technology - though these are often covered by digital inclusion/exclusion/universal barriers.

**Comment by jbuller** [comment]
**Accessible**
The status of giving all kinds of people, equivalent experiences and opportunities, despite limitations in their capabilities (leading to access needs).
Avoid using as a synonym for ‘reachable’ when referring to information, a service or people.

**Comment by sshukor** [comment]
Hi James,

Thanks for your suggestions.

Do you have any research insights supporting these suggestions? It's not always needed as we also rely on best practice but, where we can, we try to be evidence-based.

The style guide is generally - with a few early exceptions like "refugee" - about how to use terms and terms to avoid. We don't use the guide to define terms.

Keep the suggestions coming!

Cheers,

Steven Shukor
G6 Principal Content Designer
Migration and Borders Digital
Home Office Digital
Chief Operating Officer Group, Home Office


________________________________

**Reply by jbuller** [reply]
Here are examples of where ‘accessible’ is used to mean reachable/findable/available rather than inclusive
•	Publicly accessible content security
•	[Civil Service SCS Organogram](https://ukhomeoffice.sharepoint.com/sites/intranet/SitePages/Civil-Service-SCS-Organogram.aspx) ‘is accessible by all grades of civil servants’
•	[The central Wellbeing teams monthly bulletin - March 2026](https://ukhomeoffice.sharepoint.com/sites/INTR0021/SitePages/The-central-Wellbeing-teams-monthly-bulletin---March-2026.aspx) ‘It’s easily accessible from the midlands’
•	 [ team-stress-risk-assessment-guide.docx](https://ukhomeoffice.sharepoint.com/:w:/r/sites/INTR0021/_layouts/15/Doc.aspx?sourcedoc=%7B3E18F69C-7F86-4CA8-ADEC-B1C607A6CC5E%7D&file=team-stress-risk-assessment-guide.docx&action=default&mobileredirect=true&DefaultItemOpen=1) ‘stored in an easily accessible location.’
There will be many others where I doubt that inclusivity was the intention of using ‘accessible’ - because information architecture/collation is often the driving force for initiatives. 
Secondly the alternative words are less common.
